File: AN_2017_zima/l2/zad5.cpp
#include <bits/stdc++.h>
using namespace std;



int main () {
  double dx = 0.0000000000001;
  double x=1;
  while (x<2) {
    x+=dx;
    if (x * (1 / x) != 1) {
      printf ("%.25lf\n", x);
      break;
    }
  }
}

File: AN_2017_zima/l2/zad5.jl
x= Float64(1)
while true
  x= nextfloat(x)
  if x * (1/x) != 1
    println(x)
    println(bits(x))
    println(bits(1/x))
    println(x*(1/x))
    println(bits(x*(1/x)))
    break
  end
end
